deploy/roles/r-update/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Install packages required by R packages
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ item }}"
+    state: present
+    disable_gpg_check: yes
+  with_items: "{{ r_dependencies }}"
+  tags:
+    - r-update
+
+- name: Find any locked directories
+  become: yes
+  become_method: sudo
+  find:
+    paths: "{{ r_system_package_dir }}"
+    pattern: '00LOCK-*'
+    file_type: directory
+  register: locked_dirs
+  tags:
+    - r-update
+
+- name: Remove locked directories
+  become: yes
+  become_method: sudo
+  file:
+    path: "{{ item.path }}"
+    state: absent
+  with_items:
+    - "{{ locked_dirs.files }}"
+  tags:
+    - r-update
+
+#Address - No /usr/bin/Rscript error
+#- name: Install Rscript
+#  become: yes
+#  become_method: sudo
+#  yum:
+#    name: https://mirrors.sonic.net/epel/6/x86_64/epel-release-6.8.noarch.rpm
+#    state: present
+#  register: rscript_epel
+
+- include_role:
+    name: epel
+  tags:
+    - epel
+
+- name: Install R
+  become: yes
+  become_method: sudo
+  yum:
+    name: R
+    state: present
```

Comment:
This should be removed, since R is installed in the `r-core` role.  It can be assumed that `r-core` has been run at least once before `r-update` is ever run, since we don't want to include it as a dependency explicitly, which would cause it to run whenever `r-update` is run.